mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1811,6 +1811,7 @@ impl JsonRpcMultiClient for ElectrumClient {
     }
 }
 
+#[cfg_attr(test, mockable)]
 impl ElectrumClient {
```

Comment:
The error at `get_tx_height_from_rpc` can be solved easily by splitting the `ElectrumClient` implementation into multiple blocks:
```rust
// This is an implementation without `mockable` attribute
impl ElectrumClient {
  // Impl here `server_ping`, `server_version`, ..., `get_tx_height_from_rpc`, ... methods that don't require to be mockable.
}

#[cfg_attr(test, mockable)]
impl ElectrumClient {
  pub fn retrieve_headers(&self, from: u64, to: u64) -> UtxoRpcFut<(HashMap<u64, BlockHeader>, Vec<BlockHeader>)> { todo!() }
}
```